From c9774be31638456b5ec5312eab5908d50abd1667 Mon Sep 17 00:00:00 2001
From: Hyun Kwon <hyun.kwon@xilinx.com>
Date: Fri, 25 Jul 2014 16:44:56 -0700
Subject: [PATCH 0418/1566] drm: xilinx: plane: Use 'dma' for the DMA channel
 name

As VDMA dependency is being removed, make the dma channel name
more generic, and use 'dma' instead of 'vdma'. Update the documentation
with a new dma-name string.

Signed-off-by: Hyun Kwon <hyunk@xilinx.com>
Acked-by: Laurent Pinchart <laurent.pinchart@ideasonboard.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
---
 .../devicetree/bindings/drm/xilinx/xilinx_drm.txt  |    4 ++--
 drivers/gpu/drm/xilinx/xilinx_drm_plane.c          |    2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/Documentation/devicetree/bindings/drm/xilinx/xilinx_drm.txt b/Documentation/devicetree/bindings/drm/xilinx/xilinx_drm.txt
index b21d946..de126eb 100644
--- a/Documentation/devicetree/bindings/drm/xilinx/xilinx_drm.txt
+++ b/Documentation/devicetree/bindings/drm/xilinx/xilinx_drm.txt
@@ -21,7 +21,7 @@ Required properties:
 
 Required plane properties:
  - dmas: the phandle list of DMA specifiers
- - dma-names: the indentifier strings for DMAs
+ - dma-names: the indentifier strings for DMAs. The value should be "dma"
  - rgb2yuv: the phandle for rgb2ycrcb IP if used for plane
  - cresample: the phandle for chroma resampler IP if used for plane
 
@@ -38,7 +38,7 @@ vdma - [remap] - rgb2yuv - cresample - [axi2vid] - adv7511
 		planes {
 			plane0 {
 				dma = <&axi_vdma_0>;
-				dma-names = "vdma";
+				dma-names = "dma";
 				rgb2yuv = <&v_rgb2ycrcb_0>;
 				cresample = <&v_cresample_0>;
 			};
diff --git a/drivers/gpu/drm/xilinx/xilinx_drm_plane.c b/drivers/gpu/drm/xilinx/xilinx_drm_plane.c
index 820affc..1c1d20e 100644
--- a/drivers/gpu/drm/xilinx/xilinx_drm_plane.c
+++ b/drivers/gpu/drm/xilinx/xilinx_drm_plane.c
@@ -613,7 +613,7 @@ xilinx_drm_plane_create(struct xilinx_drm_plane_manager *manager,
 	plane->format = -1;
 	DRM_DEBUG_KMS("plane->id: %d\n", plane->id);
 
-	plane->dma.chan = of_dma_request_slave_channel(plane_node, "vdma");
+	plane->dma.chan = of_dma_request_slave_channel(plane_node, "dma");
 	if (IS_ERR(plane->dma.chan)) {
 		DRM_ERROR("failed to request dma channel\n");
 		ret = PTR_ERR(plane->dma.chan);
-- 
1.7.5.4

